src/sysen3/papsav.2

```diff
@@ -0,0 +1,385 @@
+	TITLE PAPSAV
+
+	; EJS (2018-12-20): Sourcee reconstructed CHANNA;RAKASH PAPSAV binary.
+        ; This reconstructed version uses macros from SYSENG; CALRET > as the original
+	; source did. The original PAPSAV shared some code with SYSEN1;SYSMSG > (which
+	; also inserted SYSENG; CALRET >). Where applicable, the source from SYSMSG was
+	; used in this reconstruction.
+
+.INSRT SYSENG;CALRET >
+
+t1=1
+t=2
+tt=3
+a=4
+.prsva=4
+b=5
+c=6
+f=7
+pt=10
+ch=11
+count=12
+.prsvz=13
+.ap.=14
+.pf.=15
+.cf.=16
+p=17
+
+tyic==2 ; tty input channel
+tyoc==1 ; tty output channel
+tyo=<.iot tyoc,> ; instruction to output a char
+
+	0		;not sure why this is needed
+
+; SYSTEM MESSAGE BUFFER CONTAINS 8-WORD ENTRIES AS FOLLOWS:
+;	ABCDEF,,ASCIZ
+;	ARG1
+;	...
+;	ARG6
+;	TIME
+; A..F = FORMAT OF ARG1..ARG6 RESPECTIVELY, CODED:
+FMTBL:	0	;0 END OF MESSAGE
+	TYPOCT	;1 FULL WORD OCTAL
+	TYPDEC	;2 FULL WD DECIMAL
+	TYPHAF	;3  " WITH COMMAS
+	TYCRLF	;4 DO CR
+	CPOPJ	;5 ?
+	TYPSIX	;6 SIXBIT
+	TYPASZ	;7 ASCIZ
+
+CPOPJ:	POPJ P,
+
+feep:   blt b,(.prsva)
+CRLF:	ASCIZ/
+/
+
+CMACMA:	ASCIZ/,,/
+
+SEP:	ASCIZ/-----
+/
+
+bletch: asciz\
+WRONG SYSTEM VERSION - REPURIFY
+\
+
+MEMHOL:	ASCIZ\
+WARNING: THERE IS A HOLE IN MEMORY
+\
+
+;;; MAIN PROGRAM
+
+PROGRAM SYSMSG
+
+	REGISTER A,B,C,F,PT,CH,COUNT
+
+	.call open,[[.uao,,tyoc],[sixbit/DSK/],[sixbit/CDATA/],[360000,,000000],[sixbit/DRAGON/]]
```

Comment:
360000,000000 should be SIXBIT/>/

Reply:
I should have added a comment.  The ">", even within sixbit/>/ breaks the .call macro.